opencti-platform/opencti-front/src/private/components/locations/regions/RegionCreation.tsx

```diff
@@ -94,7 +94,7 @@ const regionMutation = graphql`
   }
 `;
 
-interface RegionAddInput {
+export interface RegionAddInput {
```

Comment:
Why we have an export here ?

Reply:
It's a left over from my attempt to type a generified component. I will remove it.